CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.1...HEAD)
 
 ### Backward Compatibility Fixes
+- Use short array syntax `[]` instead of long `array()` - so break compatibility with PHP 5.3
```

Comment:
The heading should be 'Backwards Compatibility Breaks', but this changelog entry should be with the composer.json update to >=5.4 since it is already not allowed to be used.